aravis.spec

```diff
@@ -128,10 +122,45 @@ fi
 update-mime-database %{_datadir}/mime &> /dev/null || :
 update-desktop-database &> /dev/null || :
 
+%files
+%{_bindir}/arv-tool-%{majorversion}
+%{_bindir}/arv-fake-gv-camera-%{majorversion}
+%{_libdir}/lib%{fullname}*.so.*
+%{_libdir}/girepository-1.0/*
+%{_docdir}/%{name}/%{fullname}
+%{_datadir}/%{fullname}/*.xml
+
+%files devel
+%{_includedir}/%{fullname}
+%{_libdir}/pkgconfig/*
+%{_datadir}/gir-1.0/*
+%{_datadir}/gtk-doc/html/%{fullname}
+%{_libdir}/lib%{fullname}*.so
+
+%files static
+%{_libdir}/lib%{fullname}*.a
+
+%files -f %{fullname}.lang viewer
+%{_bindir}/arv-viewer
+%{_datadir}/%{fullname}/arvviewer.h
+%{_datadir}/%{fullname}/arvviewertypes.h
+%{_datadir}/%{fullname}/*.ui
+%{_datadir}/icons/hicolor/*/apps/*
+%{_datadir}/applications/arv-viewer.desktop
+%{_datadir}/appdata/arv-viewer.appdata.xml
+
 %files gstreamer1
 %{_libdir}/gstreamer-1.0/*
 
 %changelog
+* Tue Feb 21 2017 Mark Harfouche <mark.harfouche@gmail.com> 0.5.7
+- New upstream release
+- Enabled usb support
+- I didn't know where to put
+%{_datadir}/%{fullname}/arvviewer.h
+%{_datadir}/%{fullname}/arvviewertypes.h
+- Enabled packet-socket
+- Enabled fast-heartbeat
```

Comment:
Please remove fast-heartbeat, it is an advanced feature, and it even should not be a compile time choice.